### DISCUSSION TITLE
Showcase

### DISCUSSION THREAD

**Discussion by LaG1924** [LaG1924/AltCraft#69] [Show and tell]
Please submit AltCraft screenshots and videos here. Some will be added to README.md.

**Comment by LaG1924** [comment]
[![AltCraft YT](https://img.youtube.com/vi/LvvsEjPvCmY/0.jpg)](https://www.youtube.com/watch?v=LvvsEjPvCmY)
![ac_4](https://user-images.githubusercontent.com/12997935/124433999-cbd7a780-dd8c-11eb-812d-e89c65ffabc6.png)
![ac_3](https://user-images.githubusercontent.com/12997935/124433848-9fbc2680-dd8c-11eb-9c68-a0a9a3dcaecb.png)
![ac_2](https://user-images.githubusercontent.com/12997935/124433850-a054bd00-dd8c-11eb-97c9-e9a8c6cca32d.png)
![ac_1](https://user-images.githubusercontent.com/12997935/124433852-a0ed5380-dd8c-11eb-8291-1e18f65cc511.png)

**Comment by Grass-block** [comment]
that`s amazing!
but it sems a little bit weird in graphics.
I think you could add something like smoothLighting and blend rendering
